src/azure-cli/HISTORY.rst

```diff
@@ -38,7 +38,12 @@ Release History
 
 * az webapp up detects env folder and removes it from compressed file used for deployment
 
-**Keyvault**
+**HDInsight**
+
+* resize: Made --workernode-count/-c required
```

Comment:
The following description may be better:
`az hdinsight resize`: Make parameter `--workernode-count/-c` required

Reply:
Fixed